### ISSUE TITLE
New algorithm for rescaling % live fractions returned by scale_phenology

### ISSUE THREAD

**Issue by kpalmqui** [DrylandEcology/rSFSTEP2#224]
Currently, we rescale the % live fractions return by `scale_phenology` by rescaling the dervied values to sum to the sum of the default values in InputData_PctLive.csv.

This implementation results in some undesirable outcomes. In particularly, for sites that are warmer than the default site, the peak is always compressed, while the opposite is true for sites that are colder than the default site.

Warmer site:
![image](https://user-images.githubusercontent.com/7769797/77690086-5dd8e980-6f68-11ea-9d50-b6071ebcd6d4.png)

Colder site:
![image](https://user-images.githubusercontent.com/7769797/77690110-68937e80-6f68-11ea-9ad2-1bb666e039ad.png)

Thus, @dschlaep and I have decided to change the way the % live fractions are rescaled. 
New algorithm:
```
for each month{
 pct_live[month] = pct_live[month] * max(pct_live_default) / max(pct_live)
}
```

**Comment by kpalmqui** [comment]
Some initial Figures from current conditions and one future scenario that indicate the impact of the new implementation. More testing Figures to follow.

![p cool grass_1-2_pctlive_graph](https://user-images.githubusercontent.com/7769797/77934859-e8be2a80-726d-11ea-824c-07d5256c4134.png)

![a cool grass_1-2_pctlive_graph](https://user-images.githubusercontent.com/7769797/77934867-eb208480-726d-11ea-9c43-ecbe5129888b.png)

![sagebrush_1-2_pctlive_graph](https://user-images.githubusercontent.com/7769797/77934880-eeb40b80-726d-11ea-8460-d5703aff4c4e.png)

**Comment by kpalmqui** [comment]
We have decide to also implement this new scaling for litter fractions.

**Comment by kpalmqui** [comment]
Closed by a7c6c60
